### DISCUSSION TITLE
DNSDB Integration

### DISCUSSION THREAD

**Discussion by Mr-Lazzy** [blacklanternsecurity/bbot#779] [Ideas]
**Description**
Which feature would you like to see added to BBOT? What are its use cases?
Just want to know if BBOT has DNSDB integration

**Comment by TheTechromancer** [comment]
@Mr-Lazzy thanks for the suggestion! We don't have a DNSDB module yet, but I will add it to the todo.

Would you be interested in kick-starting this? We already have a subdomain enumeration template, so it should be a pretty quick affair.

Let me know and I can give you a quick rundown of the steps.
